### DISCUSSION TITLE
price impact too high 97.5

### DISCUSSION THREAD

**Discussion by feirt1** [beeb/pancaketrade#90] [General]
hi i have a  problem when i buy a token , in every token i buy i have this error
![msg2027416420-1652](https://user-images.githubusercontent.com/106494421/170892381-5cec00c2-a518-4fab-90e6-530d49c5eb2e.jpg)

i put this conf, before i had 0.05 and i still got errors  

[ EDITED FOR PRIVACY ]

**Comment by beeb** [comment]
If you link the contract address of that token, I can try to reproduce and have a look.

**Reply by beeb** [reply]
PS: I removed your second image as it contained a private token. Don't share your secrets section of the config file.

**Comment by beeb** [comment]
There was indeed a bug. It has been fixed in https://github.com/beeb/pancaketrade/releases/tag/v0.7.2
